.github/workflows/suite-native_develop_ci.yml

```diff
@@ -14,137 +14,34 @@ concurrency:
-        # builds and publishes the app
-        run: |
-          bundle install
-          bundle exec fastlane android develop
-
-      - uses: actions/upload-artifact@v3
+          node-version: 18.x
+          cache: yarn
+      - name: Setup EAS
+        uses: expo/expo-github-action@v8
         with:
-          name: app-apk
-          path: |
-            suite-native/app/android/**/*.apk
-
-  ios_develop:
-    runs-on: macos-12
-    environment: develop
-    timeout-minutes: 75
-    steps:
-      - name: Checkout code
-        uses: actions/checkout@v3
-
-      - name: Setup node
-        uses: actions/setup-node@v3
-        with:
-          node-version-file: ".nvmrc"
-
-      - name: Get yarn cache directory path
-        id: yarn-cache-dir-path
-        run: echo "::set-output name=dir::$(yarn config get cacheFolder)"
-
-      - name: Restore yarn cache
-        uses: actions/cache@v3
-        id: yarn-cache
-        with:
-          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
-          key: yarn-cache-folder-${{ hashFiles('**/yarn.lock', '.yarnrc.yml') }}
-          restore-keys: |
-            yarn-cache-folder-
-
-      - name: Decode files
-        env:
-          APPSTORE_CERTIFICATE_CONTENT: ${{ secrets.APPSTORE_CERTIFICATE_CONTENT }}
-        run: |
-          echo $APPSTORE_CERTIFICATE_CONTENT | base64 -d > suite-native/app/TrezorDistributionCertificate.p12
-
-      - name: Install missing Python deps (to build bcrypto lib in Node)
-        run: pip install setuptools
-
-      - name: Install dependecies
-        run: |
-          yarn install
-          yarn build:libs
-
-      - name: Expo Prebuild
-        working-directory: suite-native/app/
+          eas-version: latest
+          token: ${{ secrets.EXPO_TOKEN }}
+      - name: Install libs
+        run: yarn install
+      - name: Build on EAS
         env:
           SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
-        run: yarn expo prebuild --platform ios
-
-      - name: Ruby Setup for Fastlane
-        uses: ruby/setup-ruby@v1
-        with:
-          ruby-version: "3.2.1"
-          bundler-cache: true
-      - name: Install bundler
-        working-directory: suite-native/app/
-        run: gem install bundler
-
-      - name: Fastlane build and deploy
-        working-directory: suite-native/app/
-        env:
-          APPSTORE_KEY_ID: ${{ secrets.APPSTORE_KEY_ID }}
-          APPSTORE_KEY_ISSUER_ID: ${{ secrets.APPSTORE_KEY_ISSUER_ID }}
-          APPSTORE_KEY_FILE_CONTENT: ${{ secrets.APPSTORE_KEY_FILE_CONTENT }}
-          APPSTORE_CERTIFICATE_PASSWORD: ${{ secrets.APPSTORE_CERTIFICATE_PASSWORD }}
-          TMP_KEYCHAIN_NAME: ${{ secrets.TMP_KEYCHAIN_NAME }}
-          TMP_KEYCHAIN_PASSWORD: ${{ secrets.TMP_KEYCHAIN_PASSWORD }}
-
-        # builds and publishes the app
-        run: |
-          bundle install
-          bundle exec fastlane ios develop
+        run: eas build
```

Comment:
I would prefer to split android and ios builds to separate steps or just use platform all?

Reply:
Platform all is way to go

Reply:
I also believe this is better, but let's discuss. 

- you don't see build progress on the github action (not even result due to the `--no-wait` flag. I guess the result could be added, but that also makes us pay more for the github runners + expo runners, right? 
- on expo, the builds are separated so they're taken as 2 separate builds

Reply:
Yes, but I want to use `--auto-submit` for iOS, but not for android (that is handled by `upload-to-firebase.sh`)

Reply:
However for production, platform all is definitely a way to go

Reply:
https://github.com/trezor/trezor-suite/pull/10898/commits/e19d2e394566740d9cc5b6deea48384172dfc5a2